.github/workflows/merge_stage_test.yml

```diff
@@ -33,7 +33,9 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
       - name: Upgrade pip
-        run: pip install pip --upgrade
+        run: |
+          pip install pip --upgrade
+          pip install wheel
```

Comment:
please add this reference 
https://github.com/pypa/pip/blob/1463081f10de6bfad81afe0d68272e7c3bedbadf/src/pip/_internal/wheel_builder.py#L94-L97